src/ClaimService.Business/Features/Categories/Commands/Edit/EditCategoryValidator.cs

```diff
@@ -0,0 +1,71 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ClaimService.Business.Shared.Enums;
+using LT.DigitalOffice.Kernel.Validators;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ClaimService.Business.Features.Categories.Commands.Edit;
+
+public class EditCategoryValidator : BaseEditRequestValidator<EditCategoryRequest>, IEditCategoryValidator
+{
+  private async Task HandleInternalPropertyValidation(
```

Comment:
maybe void?

Reply:
Nope. CustomAsync() requires Task to be returned

Reply:
Maybe it will work if use Custom()